src/MetaMorpho.sol

```diff
@@ -327,7 +327,10 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             if (!seen[i]) {
                 Id id = withdrawQueue[i];
 
-                if (MORPHO.supplyShares(id, address(this)) != 0 || config[id].cap != 0) {
+                // The following call cannot revert.
```

Comment:
Wdyt about specifying in the natspecs instead:
`/// @dev Should not revert.`

Reply:
we can do both I think